website_sale_infinite_scroll/models/product_template.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.model
+    def create_dummy_products(self):
+        return self.create(
+            [
+                {"name": "product {num}".format(num=x), "website_published": True}
```

Comment:
Maybe it makes sense to just reduce the number of products on the shop page?

Reply:
Thank you, yes, everything is correct, but this is to test all the functionality, I will transfer it to the demo.